x-pack/plugins/infra/server/lib/alerting/metric_threshold/register_metric_threshold_alert_type.ts

```diff
@@ -17,7 +18,10 @@ const oneOfLiterals = (arrayOfLiterals: Readonly<string[]>) =>
       arrayOfLiterals.includes(value) ? undefined : `must be one of ${arrayOfLiterals.join(' | ')}`,
   });
 
-export async function registerMetricThresholdAlertType(alertingPlugin: PluginSetupContract) {
+export async function registerMetricThresholdAlertType(
+  alertingPlugin: PluginSetupContract,
+  libs: InfraBackendLibs
```

Comment:
`libs` is unused in this alert type, and on my end it doesn't look like we need to add this arg to get it to pass type check. Am I missing something?

Reply:
This was mostly laziness to deal with:

```
registerFns.forEach(fn => {
  fn(alertingPlugin, libs);
});
```

it was easier just to pass them both the `libs`.

As the work Philip and I are doing has some overlap here, where we've both refactored the source configuration code and allowed access to `libs` in executors, and Philip has refactored this `forEach` line, I think it would be easier just to leave this for this merge. And deal with the small conflict that will arise in Philip's branch.

Reply:
// CC @phillipb

Reply:
Sounds good, we can do that.